lib/generators/light_service/generator_utils.rb

```diff
@@ -0,0 +1,45 @@
+module LightService
+  module Generators
+    module GeneratorUtils
+      def make_nested_dir(dir)
+        FileUtils.mkdir_p(dir)
+      end
+
+      def supported_test_frameworks
+        %i[rspec]
+      end
+
+      def test_framework_supported?
+        supported_test_frameworks.include? test_framework
+      end
+
+      # Don't know a better way to get to this value, unfortunately.
+      def test_framework
+        # Rails.application.config.generators.options[:rails][:test_framework]
+        # When/if Minitest is supported, this will need to be updated to detect
+        # the selected test framework, and switch templates accordingly
+        :rspec
+      end
+
+      def must_gen_tests?
+        options.tests? && test_framework_supported?
+      end
+
+      # rubocop:disable Metrics/AbcSize
+      def create_required_gen_vals_from(name)
```

Comment:
I'd shy away from abbreviations. Consider renaming this method to `create_required_generator_values_from()`.